### PR TITLE
Add helicopters and move E3 prerequisites to campaign-rules.yaml

### PR DESCRIPTION
**Pull request by Smittytron**
Split off from #17963. Adding campaign specific helicopter prereqs as we're getting into late-tech missions. Also moving e3 override since almost soviet mission needs it.

### CODE REVIEW

File: mods/ra/maps/allies-03a/rules.yaml [hunk 1]
```diff
@@ -175,10 +175,6 @@ KENN:
 	Buildable:
 		Prerequisites: ~disabled
 
-E3:
-	Buildable:
-		Prerequisites: ~disabled
```
Comment (Mailaender) [inline mods/ra/maps/allies-03a/rules.yaml:180]: Removing this enables them for this mission. Is that wanted?

Comment (Smittytron) [reply mods/ra/maps/allies-03a/rules.yaml:180]: It's moved to ~tent, which are unavailable in this mission.

**Review by cjshmyr — DISMISSED** [review]
Works as intended

**Comment by abcdefg30** [comment]
As I've said before, I'm still not convinced we should give Soviets any helicopters. Until then :-1:

**Comment by cjshmyr** [comment]
Removing my +1 for now until there is consensus on campaign helicopters. Rocket soldier changes were fine.

Largely our missions follow OpenRA balance over the originals, examples of this:
- Fog of war
- Aircraft behavior
- Redeployable MCV
- Build radius
- Tech tree for structures and units

Funny enough, we contradict this at least once:
- Soviets do not have rocket soldiers

My biggest concern after reading the discussion on discord today is causing a rift in campaign progress -- I'm not picking sides here, so here's two points.

1) Adding soviet hinds is similar to removing rocket soldiers from soviets, matches the original balance.
2) Removing soviet hinds (using Yaks instead) is in line with our current mission philosophy.

We are approaching missions with soviet helicopters so it's a good idea to raise concerns now.

**Comment by abcdefg30** [comment]
> Soviets do not have rocket soldiers

Which is Imho an inconsistency we should look to fix. (And what I would have never introduced myself.)
:+1: to your second point.

**Comment by Smittytron** [comment]
> As I've said before, I'm still not convinced we should give Soviets any helicopters.

This throws a wrench in my plans for most of the remaining missions. We need a path decided sooner than later to keep campaign progress on track.

I'm on Team Hind on this one. As stated in discord, my approach to missions is to only deviate from the original when OpenRA clearly does something better. Balance doesn't really justify the deviation for me; campaign missions are never balanced as you almost always start from nothing vs an established opponent.

I also view this as a good way to reintroduce the hind, which as an original asset, should be in the game in some capacity.

 The following missions should have Soviet hinds: Allies 09, Allies 10, Allies 11, Allies 12, Allies 14, Soviet 11, Soviet 12, Soviet 13, Soviet 14, Absolute MADness, Brothers in Arms, Caught in the Act, Deux ex Machina, Don't Drink the Water, Fall of Greece 2, Grunyev Revolution, Harbor Reclamation, Sarin Gas 3, Siberian Conflict 2, Siberian Conflict 3, Testing Grounds.

I'll try to focus on missions that don't have hinds until we have a final decision, but there aren't many of those left.

**Comment by pchote** [comment]
IMO our primary focus should be to get the missions in at all so that we can then build the proper campaign flow and have *something* that players can work through.

We already know that most missions will need to be adjusted to rebalance difficulty levels, properly use OpenRA-RA's tech tree and features, etc. Why can't we save the hind removal until then?

**Comment by abcdefg30** [comment]
I don't even know where to begin. I thought we went through all of this yesterday on Discord, but now I suppose not.

Apparently one thing we did agree on is that we have to adapt the missions to our tech tree and gameplay (as was the approach in the beginning when I started). That makes this even more baffling, since it is a step back, adding to inconsistencies with the argument that “someone else” will fix everything later. I really do wonder who that would be, since if that is really how we want to approach things I’ll stop doing anything mission related (again). (And Smitty will probably not fix it either, considering how he doesn’t want to do things this way the first time around.)

> IMO our primary focus should be to get the missions in at all so that we can then build the proper campaign flow and have something that players can work through.

This statement strikes me as ironic, since we probably would be much further (maybe even done) if it weren’t for motivation killers like this.

Tl;dr: Do what you must, I suppose in the end this is just another drop in the ocean. I just wanted to be open about not going to be much help and maybe even continue to actively :-1: PRs. (If I don’t use this as reason to finally resign from the project, I guess; we’ll see.)

**Comment by Smittytron** [comment]
Sorry, I didn't recognize the issues this would dredge up.
